### PR TITLE
EJDOTNETCORE-4659 Added converting XPS to PDF md file.

### PR DESCRIPTION
**Pull request by sameerkhan001**
Hi All,

Added Converting XPS to PDF md file in UG.

Regards,
Sameerkhan N

### CODE REVIEW

File: File-Formats/PDF/Converting-XPS-to-PDF.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+description: This section explains how to converting XPS document to PDF document by using Syncfusion .NET PDF library.
+platform: file-formats
+control: PDF
+documentation: UG
+---
+# Converting XPS document to PDF
+
+The XPS (XML Paper Specification) document format is a fixed document format which consists of structured XML markup that defines the layout of a document and the visual appearance of each page, along with rendering rules for distributing, archiving, rendering, processing and printing the documents.
+
+The following code example explains how to converting XPS document to PDF using [XPSToPdfConverter](https://help.syncfusion.com/cr/file-formats/Syncfusion.XPS.XPSToPdfConverter.html) class.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Converting-XPS-to-PDF.md:12]: The following code example illustrates how to convert XPS to PDF ...

File: File-Formats/PDF/Converting-XPS-to-PDF.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+The below code illustrates how to convert XPS to PDF.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Converting-XPS-to-PDF.md:14]: Remove this line

File: File-Formats/PDF/Converting-XPS-to-PDF.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+//Essential PDF supports converting XPS document to PDF only in Windows Forms, WPF, ASP.NET, ASP.NET MVC and UWP platforms.
+
+{% endhighlight %}
+
+{% endtabs %}
+
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PDF-Examples/tree/master/Document%20conversion/Converting-XPS-to-PDF-document).
+
+N> Essential PDF supports converting XPS to PDF with [Syncfusion.XpsToPdfConverter.Net.Core](https://www.nuget.org/packages/Syncfusion.XpsToPdfConverter.Net.Core) package reference in .NET Core application.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Converting-XPS-to-PDF.md:99]: The Syncfusion .NET PDF library supports converting XPS to PDF with [Syncfusion.XpsToPdfConverter.Net.Core](https://www.nuget.org/packages/Syncfusion.XpsToPdfConverter.Net.Core) NuGet package reference in ASP.NET Core application.

File: File-Formats/PDF/Converting-XPS-to-PDF.md (hunk 1 above)
Comment (sameerkhan001) [reply File-Formats/PDF/Converting-XPS-to-PDF.md:12]: Resolved

File: File-Formats/PDF/Converting-XPS-to-PDF.md (hunk 2 above)
Comment (sameerkhan001) [reply File-Formats/PDF/Converting-XPS-to-PDF.md:14]: Removed

File: File-Formats/PDF/Converting-XPS-to-PDF.md (hunk 3 above)
Comment (sameerkhan001) [reply File-Formats/PDF/Converting-XPS-to-PDF.md:99]: Changed

**Review by praveenHari24 — APPROVED** [review]

**Pull request merged**